tests/test_widget_child_moving.py

```diff
@@ -59,9 +59,9 @@ async def test_widget_move_child() -> None:
             container = Widget(*widgets)
             await pilot.app.mount(container)
             container.move_child(child, before=target)
-            assert container.children[0].id == "widget-1"
-            assert container.children[1].id == "widget-0"
-            assert container.children[2].id == "widget-2"
+            assert container._nodes[0].id == "widget-1"
+            assert container._nodes[1].id == "widget-0"
+            assert container._nodes[2].id == "widget-2"
```

Comment:
I see a lot of these in the tests, so picking one at random to ask the question: why have all the tests been changed to use `_nodes` rather than `children`? I see this in the internal code too and I'm assuming there it's just to leave out one level of indirection, just to make the repeated calls a wee bit faster, but if (as it seems to me) `children` is a thin wrapper around `_nodes`, as a public interface, shouldn't the tests be testing the public interface?

Reply:
`children` was renamed to `_nodes` in this PR. So these are testing the same things as before.

 I probably should have additional tests for the new `children` attribute.